### PR TITLE
added card component.

### PR DESCRIPTION
**Pull request by zoyatanweer**

### CODE REVIEW

File: css/common.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
-@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;900&display=swap');
+/* @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;900&display=swap'); */
```
Comment (zoef-yt) [inline css/common.css]: remove commented code.

File: components/card/cards.css [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+}
+.card-container-vertical{
+    display: flex;
+    flex-direction: row;
+}
+.iframe-card{
+    margin-left:10rem;
+    margin-top:1.5rem;
+}
+
```
Comment (zoef-yt) [inline components/card/cards.css:14]: some spacing inconsistentcy.

File: components/card/cards.css [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    flex-direction: column;
+    align-items: left;
+    margin-bottom:3rem;
+  }
+ 
+.card-img {
+    height: 23rem;
+    width:23rem;
+    border-radius: 0.5rem 0.5rem 0 0;
+    /* margin-top:0; */
```
Comment (zoef-yt) [inline components/card/cards.css]: remove commented code.

Comment (zoef-yt) [inline components/card/cards.css]: your images are squeezed. Use  `object-fit:cover` inside img class.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: components/card/cards.css [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    padding:4px;
+      position: absolute;
+    left:31rem;
+}
+.dismiss-btn{
+    color:#6c757d;
+    font-size:2rem;
+    position:absolute;
+    left: 51.5rem;
+    margin-top:0.5rem;
```
Comment (iRohitGaur) [inline components/card/cards.css]: use px if not changing this?

**Review by iRohitGaur — APPROVED** [review]
LGTM! Good to merge 👍🏻

File: components/card/cards.css (hunk 3 above)
Comment (zoyatanweer) [reply components/card/cards.css]: thanks for the suggestion. It now looks  much better.

File: components/card/cards.css (hunk 4 above)
Comment (zoyatanweer) [reply components/card/cards.css]: thanks for pointing out, changing into px.

**Review by zoef-yt — APPROVED** [review]
Looks good. Good to merge.

**Pull request merged**